File: src/main/java/com/example/demo/entity/ExampleEntity.java

package com.example.demo.entity;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "examples")
@Data
@NoArgsConstructor
public class ExampleEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String name;

    @Column
    private String description;
}

File: src/main/kotlin/com/example/demo/entity/ExampleEntity.kt
package com.example.demo.entity

import jakarta.persistence.*

@Entity
@Table(name = "examples")
class ExampleEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null

    @Column(nullable = false)
    var name: String = ""

    @Column
    var description: String? = null
}
